src/cwts/networkanalysis/run/FileIO.java

```diff
@@ -0,0 +1,396 @@
+package cwts.networkanalysis.run;
+
+import cwts.networkanalysis.Clustering;
+import cwts.networkanalysis.Layout;
+import cwts.networkanalysis.Network;
+import cwts.util.DynamicDoubleArray;
+import cwts.util.DynamicIntArray;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Arrays;
+
+/**
+ * Utility functions for file I/O.
+ *
+ * <p>
+ * All methods in this class are static.
+ * </p>
+ *
+ * @author Ludo Waltman
+ * @author Nees Jan van Eck
+ * @author Vincent Traag
+ */
+public class FileIO
```

Comment:
Should we perhaps add a private constructor in order to prevent the class from being instantiated?

Reply:
Excellent point! Changed.